functions/Get-DbaInstallDate.ps1

```diff
@@ -0,0 +1,188 @@
+function Get-DbaInstallDate
+{
+<#
+.SYNOPSIS
+Returns the install date of SqlServer and Windows Server, depending on what is passed. 
+	
+.DESCRIPTION
+By default, this command returns for each SQL Server instance passed in:
+SQL Instance install date, formatted as a string
+Hosting Windows server install date, formatted as a string
+	
+.PARAMETER SqlServer
+The SQL Server that you're connecting to.
+
+.PARAMETER SqlCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER WindowsCredential
+Credential object used to connect to the SQL Server as a different user
+
+.PARAMETER SqlOnly
+Excludes the Windows server information
+
+.PARAMETER WindowsOnly
+Excludes the SQL server information
+
+.NOTES
+Tags: CIM 
+Original Author: Mitchell Hamann (@SirCaptainMitch), mitchellhamann.com
+	
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
```

Comment:
Check the [Get template](https://github.com/sqlcollaborative/dbatools-templates/blob/master/Get-DbaSmoObject.ps1#L23) we have changed the license section to be much shorter.